tests/test_http_response.py

```diff
@@ -14,9 +14,12 @@ def normalize_headers(self, headers):
         """
         return sorted(
             [
+                (b"server", b"daphne"),
+            ]
+            + [
                 (name.lower(), value.strip())
                 for name, value in headers
-                if name.lower() != b"transfer-encoding"
+                if name.lower() != b"transfer-encoding" and name.lower() != b"server"
```

Comment:
When comparing against multiple values, use `in` or `not in` as appropriate.

```suggestion
                if name.lower() not in (b"transfer-encoding", b"server")
```